src/App.js

```diff
@@ -1,25 +1,17 @@
 import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import MainPage from './components/mainpage/MainPage';
+import LifeStyle from './components/lifestyle/LifeStyle';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app-container">
+      <Router>
+        <Routes>
+          <Route path="/" element={<MainPage />} />
+          <Route path="/lifestyle" element={<LifeStyle />} />
+        </Routes>
+      </Router>
```

Comment:
As per project requirements, you should have the following path in the Navigation panel.

`<Routes>
      <Route path="/" element={<Motors/>} />
      <Route path="/reserves" element={<Reserve />} />
      <Route path="/my-reservations" element={<MyReservations/>} />
      <Route path="/add-motor" element={<AddMotor/>} />
      <Route path="/delete-motor" element={<DeleteMotor />} />
      <Route path="*" element={<NoMatch />} />
    </Routes>
  </Router>`
Please kindly create those components and link them to the navigation panel

Reply:
Hi @binodbhusal 
Thanks for review, done